src/filesystem.cpp

```diff
@@ -857,45 +824,77 @@ size_t LocalPath::getLeafnameByteIndex(const FileSystemAccess& fsaccess) const
 bool LocalPath::backEqual(size_t bytePos, const LocalPath& compareTo) const
 {
     auto n = compareTo.localpath.size();
-    return bytePos + n == localpath.size() && !memcmp(compareTo.localpath.data(), localpath.data() + bytePos, n);
+    return bytePos + n == localpath.size() && !localpath.compare(bytePos, n, compareTo.localpath);
 }
 
 LocalPath LocalPath::subpathFrom(size_t bytePos) const
 {
-    return LocalPath::fromLocalname(localpath.substr(bytePos));
+    LocalPath result;
+    result.localpath = localpath.substr(bytePos);
+    return result;
```

Comment:
why the change may I ask? :relaxed:

Reply:
fromLocalname doesn't exist anymore - we could have used fromPlatformEncoded, but I think this way makes it clear that there are no conversions going on, it's just a portion of that exact string.